client/src/document/toolbar/flaws.tsx

```diff
@@ -1,6 +1,9 @@
 import React, { useEffect, useReducer, useRef } from "react";
 import { useLocation, useNavigate } from "react-router-dom";
 import useSWR from "swr";
+import { annotate, annotationGroup } from "rough-notation";
+import { RoughAnnotation } from "rough-notation/lib/model";
```

Comment:
This line, VSCode inserted it for me simply by me using `: RoughNotation[]` in the instansiation of the `const annotations = ...` variable!